### DISCUSSION TITLE
Filter hostgroups and servicegroups

### DISCUSSION THREAD

**Discussion by zjelcic** [chriscareycode/nagiostv-react#72] [Ideas]
Hello,
can you add filtering the host and services groups so we can choose multiple host groups and service groups to monitor and exclude those that are not that important?
Thanks

**Comment by chriscareycode** [comment]
Off the top of my head I am not sure if the Nagios CGI API or the LiveStatus API support sending multiple hostgroups up. When I get time to work on adding servicegroups I will test and see if this is possible

**Comment by chriscareycode** [comment]
I tried sending multiple hostgroups or multiple service groups to the Nagios CGI API and it only seems to support one at a time.
Service Groups are added and the feature will go out on the next release. We can iterate from there if new information is discovered.

**Reply by zjelcic** [reply]
Thank you very much. This will he very helpfull. If something comes on my mind I will continue on this topic.
Best regards
